frontend/src/common-components/navigation/Navbar.tsx

```diff
@@ -39,7 +39,7 @@ export const Navbar = () => {
           </Flex>
         </Flex>
         <Flex flex={{ base: 1, md: 0 }} mr={{ base: -2, md: 0 }} justify="flex-end">
-          <ColorModeSwitcher />
+          {!config?.components?.style?.forceDarkMode && <ColorModeSwitcher />}
```

Comment:
The ColorModeSwitcher component already has a similiar condition (but for a different field of config), we should put them both in the same place.